Sources/SwiftDocC/Semantics/Symbol/DeprecationSummary.swift

```diff
@@ -24,7 +24,7 @@ import Markdown
 /// }
 /// ```
 ///
-/// You can use the `@DeprecationSummary` directive top-level in both articles and documentation extension files.
+/// You can use the `@DeprecationSummary` directive top-level in articles, documentation extension files, or documentation comments.
```

Comment:
FYI: you need to run `swift run generate-symbol-graph` for this change to be reflected in the published documentation.

Reply:
Ah yes, thanks. I updated this in the latest commit.

Reply:
It would also be good to add an "Earlier Versions" aside, like on [the @Available documentation](https://www.swift.org/documentation/docc/available) to say that before Swift-DocC 6.2 (or 6.1 if we plan on cherry-picking this), @DeprecationSummary isn't supported in documentation comments.

Reply:
Good idea. Updated, thanks! I also realised that `CustomMetadata` is actually intentionally hidden from documentation (although I'm not sure why), so I removed its reference in this document.